src/_component/menu/menu.pug

```diff
@@ -0,0 +1,15 @@
+.menu-wrap
+  input.toggler(type='checkbox' id="toggler")
+  i.icon__menu
+  .menu 
+    ul
+      each item in data.burger.item 
+        li
+          a(href=`${item.link}`)=item.title
```

Comment:
you don't need to use backticks when there is only the value
`a(href=item.link)= item.title`